cartesian_torque_controller:
    type: pick_and_place/CartesianTorqueController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    publish_rate: 10.0
